packages/gatsby-plugin-onesignal/README.md

```diff
@@ -0,0 +1,51 @@
+# @vtex/gatsby-plugin-onesignal
+
+A simple Gatsby plugin to add [OneSignal](https://onesignal.com) integration into your Gatsby site.
+
+## Install
+
+To install this plugin, simply run:
+
+```
+yarn add @vtex/gatsby-plugin-onesignal
+```
+
+or
+
+```
+npm install @vtex/gatsby-plugin-onesignal
+```
+
+## Configuration options and defaults
+
+To make this plugin easy to use and fast to setup, we're using the "Typical Site" integration that OneSignal offers along with customized service worker paths and filenames, to avoid any conflicts with existing service workers your site may already have. 
```

Comment:
```suggestion
To make this plugin easy to use and fast to setup, we're using the "Typical Site" integration that OneSignal offers along with customized service worker paths and filenames, to avoid any conflicts with existing service workers on your site may already have. 
```

Reply:
In this case, I don't think `on` fits in here 🤔